docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -1,57 +1,57 @@
 # Integrate with SAP Cloud Logging
 
-## Overview
-
 | Category| |
 | - | - |
 | Signal types | logs, traces, metrics |
 | Backend type | third-party remote |
 | OTLP-native | yes for traces and metrics, no for logs |
 
-Learn how to define LogPipelines and TracePipelines to ingest application and access logs as well as distributed trace data in instances of [SAP Cloud Logging](https://help.sap.com/docs/cloud-logging?locale=en-US&version=Cloud).
-
-SAP Cloud Logging is an instance-based and environment-agnostic observability service that builds upon OpenSearch to store, visualize, and analyze logs, metrics, and traces.
-
-![setup](./../assets/sap-cloud-logging.drawio.svg)
+Learn how to configure the Telemetry module to ingest application and access logs as well as distributed trace data and metrics in instances of SAP Cloud Logging.
```

Comment:
Why not keeping the link to the CLoudLogging entry page? Shouldn't that be part of the abstract already as it is all about CloudLogging here

Reply:
Also the small categorization of what CloudLogging is about is useful..

Reply:
I see, you moved the overview down, still the link is lost, why not keeping it in the abstract?

Reply:
> Also the small categorization of what CloudLogging is about is useful..

I think Cloud Logging should do this in their docs, not we in ours

Reply:
You have a bunch of integration guides. The users goes through the guides and need to always click the external link to understand what this is about? Just a small classification of the backend please, nothing more :)

Reply:
It's in the Overview section, line 44 :)